### DISCUSSION TITLE
Support bib files

### DISCUSSION THREAD

**Discussion by Idobenhamo** [artempyanykh/marksman#238] [Ideas]
Support for linking to literature from bib files could be amazing. (also for pdf).
Maybe this is in the roadmap of the project?
Thank you for your AWESOME lsp!

**Comment by artempyanykh** [comment]
Thanks for the kind words @Idobenhamo! There's a feature request for bibtex support: https://github.com/artempyanykh/marksman/issues/189 I personally would like to see this feature added but am currently short on time (although, happy to take PRs :) )
